lib/const.rb

```diff
@@ -107,15 +107,15 @@
 
 # Local constants for contributors.
 repo_root = `git rev-parse --show-toplevel 2> /dev/null`.chomp.to_s
-if repo_root.empty? || File.basename(repo_root) != 'chromebrew'
+if repo_root.empty?
   Dir.chdir '../..' do
     repo_root = `git rev-parse --show-toplevel 2> /dev/null`.chomp.to_s
   end
 end
 CREW_LOCAL_REPO_ROOT = repo_root
 CREW_LOCAL_REPO_BASE = CREW_LOCAL_REPO_ROOT.empty? ? '' : File.basename(CREW_LOCAL_REPO_ROOT)
 CREW_LOCAL_MANIFEST_PATH = if ENV['CREW_LOCAL_MANIFEST_PATH'].to_s.empty?
-                             CREW_LOCAL_REPO_BASE == CREW_GITHUB_ACCOUNT ? "#{CREW_LOCAL_REPO_ROOT}/manifest" : ''
+                             "#{CREW_LOCAL_REPO_ROOT}/manifest"
                            else
                              ENV.fetch('CREW_LOCAL_MANIFEST_PATH', nil)
```

Comment:
The goal is to have a place where the manifest files can be put after a build, where they won't be overwritten on a crew update, right?

Maybe it is better to have an error message if `CREW_LOCAL_REPO_BASE` is empty to have the user specify a `CREW_LOCAL_MANIFEST_PATH` to put those files?

Reply:
This is simply for users running `crew build`. imo, the odds that they then unpack and install those builds, making it so the manifests are then relevant, are low. This PR just aims to reduce the barrier to entry and have `crew build <package>` "just work". (Also, I have a faint memory of being unable to install packages from source due to this issue, which would just affect regular users and is something we probably want to avoid)

Reply:
I think @Zopolis4 is correct.  There is no need for the conditional here since we already set `CREW_GITHUB_ACCOUNT`.